client/control/actions/authActions.js

```diff
@@ -0,0 +1,30 @@
+import { BASE_API_URL, API_REQUEST_SETTINGS } from '../constants.js';
+import { showNotification } from './ControlActions.js';
+
+export const AUTH_TOKEN_REQUEST_IN_PROGRESS = 'AUTH_TOKEN_REQUEST';
+export const AUTH_TOKEN_REQUEST_COMPLETE = 'AUTH_TOKEN_REQUEST_SUCCESS';
```

Comment:
I personally don't love var names THAT_LOOK_LIKE_THIS, generally I'll do something like:

```
const authTokenRequest = {
  inProgress: 'AUTH_TOKEN_REQUEST',
  success: 'AUTH_TOKEN_REQUEST_SUCCESS',
};
// ...
..authTokenRequest.inProgress..
```
But this may be more of a personal preference thing